crates/compiler/builtins/bitcode/src/dec.zig

```diff
@@ -463,6 +463,10 @@ pub const RocDec = extern struct {
         return RocDec{ .num = out };
     }
 
+    pub fn log(self: RocDec) RocDec {
+        return fromF64(@log(self.toF64())).?;
```

Comment:
Not necessary for this PR, but I just realized that we should probably have a version of `fromF64` that just calls `roc_panic` if it fails instead of generating a zig `orelse unreachable`. If this happens, we either should properly panic or potentially recover in the `log` and similar functions.